stable/database/files/nuosm

```diff
@@ -508,311 +437,85 @@ log "archiveId=$myArchive; DB=$DB_NAME; hostname=$HOSTNAME"
 atomCount=$( ls -l $DB_DIR/*.atm 2>/dev/null | wc -l )
 catalogCount=$( ls -l $DB_DIR/*.cat 2>/dev/null | wc -l )
 
-if isRestoreRequestSupported && [ -z "$legacy_restore_requested" ]; then
```

Comment:
Feel free to skim the rest of this file. The following changes are done:
- The code in the _if_ block just has an indentation change.
- The code in the _else_ block is removed.

Reply:
Clicking the "Diff view" settings and selecting "Hide whitespace" helps.